From 55a489a7885608158f66823686d89eb7cf54d5a9 Mon Sep 17 00:00:00 2001
From: Nikolai Kondrashov <Nikolai.Kondrashov@redhat.com>
Date: Tue, 2 Dec 2014 16:43:49 +0200
Subject: [PATCH 2/2] man: Remove references to naslist and clients

Remove references to non-existent naslist and clients manpages as they
are confusing.
---
 man/man5/clients.conf.5 | 8 +++-----
 man/man5/dictionary.5   | 1 -
 man/man5/users.5        | 1 -
 3 files changed, 3 insertions(+), 7 deletions(-)

diff --git a/man/man5/clients.conf.5 b/man/man5/clients.conf.5
index f9207d1..6c6b3ee 100644
--- a/man/man5/clients.conf.5
+++ b/man/man5/clients.conf.5
@@ -8,9 +8,9 @@ file contains definitions of RADIUS clients.
 .PP
 The information in this file overrides any information provided in
 the deprecated 
-.BR clients (5)
+.BR clients
 and 
-.BR naslist (5)
+.BR naslist
 files.
 .PP
 The file format is the same as that used for
@@ -39,9 +39,7 @@ that form is deprecated.
 .I /etc/raddb/radiusd.conf
 .SH "SEE ALSO"
 .BR radiusd (8),
-.BR radiusd.conf (5),
-.BR clients (5),
-.BR naslist (5)
+.BR radiusd.conf (5)
 
 .SH AUTHOR
 FreeRADIUS is authored by the FreeRADIUS team. 
diff --git a/man/man5/dictionary.5 b/man/man5/dictionary.5
index 6b60602..d685081 100644
--- a/man/man5/dictionary.5
+++ b/man/man5/dictionary.5
@@ -176,7 +176,6 @@ the attribute number should be used instead.
 .I /usr/share/freeradius/dictionary.*
 .SH "SEE ALSO"
 .BR radiusd (8),
-.BR naslist (5),
 .BR RFC2865,
 .BR RFC2866,
 .BR RFC2868
diff --git a/man/man5/users.5 b/man/man5/users.5
index ed8f471..c9aef82 100644
--- a/man/man5/users.5
+++ b/man/man5/users.5
@@ -243,7 +243,6 @@ entries that set reply attributes.
 .BR radclient (1),
 .BR radiusd (8),
 .BR dictionary (5),
-.BR naslist (5)
 
 .SH AUTHOR
 The FreeRADIUS team.
-- 
2.1.3

